src/components/HomeSection.tsx

```diff
@@ -69,13 +68,7 @@ export const HomeSection = ({
 
   return (
     <>
-      <TitleContainer>
-        <Touchable onPress={navigate}>
-          <Title accessibilityLabel={`${title} (Überschrift) (Taste)`}>{title}</Title>
-        </Touchable>
-      </TitleContainer>
-      {device.platform === 'ios' && <TitleShadow />}
-
+      <SectionHeader title={title} onPress={navigate} />
```

Comment:
this can be done on much more places across the code base

Reply:
yes. i have had it on my list for march for quite some time now 💪